CHANGELOG.md

```diff
@@ -0,0 +1,334 @@
+
+All updates to Carbon Mapper data are assigned a version number for each product level, with a detailed change log summarizing modifications and documenting the state of the corresponding algorithms. Each version reflects rigorously vetted algorithm improvements to enhance accuracy and reduce observation uncertainty. Our Algorithm Theoretical Basis Document contains algorithm justification for our most recent algorithms and citations with detailed overviews of our product levels and algorithms associated with each product level.
+
+We periodically reprocess historical data to ensure that past datasets benefit from the most recent, validated, and reliable algorithms. Previous versions of historical data remain accessible to users who may need them for specific applications or comparisons. Each historical reprocessing builds on the strengths of our existing algorithms, further enhancing data accuracy and refining precision to provide even more reliable emission estimates.
+
+* The Summary table has an overview of published data versions.
+* The Detailed Collection Impact Notes thoroughly explain how algorithms changed in a specified collection and what instruments and gases are impacted.
+* The FAQ section answers questions about how to use this change log and how to identify what version of a product you are looking at.
+
+# Summary Table
+Note: Version 2 is guaranteed to include the changes listed under version 2 products. Version 1 may include incremental changes between v1 and v2. 
+
+| **Version**             |    **High Level Notes**     | **Date Available** | **Processing SW**         |
+|:------------------------|:---------------------------:|:-------------------|:--------------------------|
+| jpl                     |    Data processed by JPL    | Aug 2016           | NA                        |
+| v0                      |   Initial data algorithms   | March 2023         | No version data available |
+| [v1 (j001^)](#version1) |                             | July 2023          | 0.4.0<                    |
+| [v2 (j002^)](#version2) | Visualization products only | Jan 2024           | 0.21.0<                   |
+| [v3 (j3^)](#version3)   |                             | Feb 2025           | 3.0.0<                    |
+
+**^** versions that begin with j, were processed by JPL using the same algorithms as the associated v<#> version.
+
+# Detailed Version Notes
+Refer to the ATBD for an in depth scientific justification of algorithms. The L2B ATBD includes details about filter types (MF/MFA/MFAL) and Unit Absorption Spectrum. The L3/L4 ATBD includes details about dynamic vs static retrievals, simple vs concentric circles IME, and maximum Fetch)
+
+**NOTE**: **<span style="color:red"> **bolded text** </span>** indicates changes to algorithms in current version
+
+# Version 3 <a name="version3"></a>
+
+## Quantification Products
+### TAN 
```

Comment:
Should we spell out these instrument names and make it clear that they're instruments?  Readers might not recognize them for what they are.